README.md

```diff
@@ -1,2 +1,2 @@
 # data-analysis
```

Comment:
The README got updated before you did this, if you could pull it, then recommit, it would help a lot with merge conflicts :)